### PR TITLE
Remove all index.ts barrel file and all b2b component imports in b2c mode

### PR DESCRIPTION
**Pull request by aChanEP**
Description:

https://elasticpath.atlassian.net/browse/RS-802
https://elasticpath.atlassian.net/browse/RS-790

Using direct path to components for imports instead of using a barrel index.ts file.  

Removal of this makes import consistent across all components (some components were using the barrel file to import and others relative links).

We have increased speed slightly with this PR as all b2b components are not loaded in b2c mode now with the removal of the index.ts.  Components are only loaded if the container is loaded in a route.

Linting:

- [x] No linting errors

Tests:

- [ ] E2E tests (npm test run with `e2e`)
- [x] Manual tests

Documentation:

- [ ] Requires documentation updates

### CODE REVIEW

File: src/containers/WishListsPage.tsx [hunk 1]
```diff
@@ -22,11 +22,10 @@
 import React from 'react';
 import intl from 'react-intl-universal';
 import { RouteComponentProps } from 'react-router-dom';
-import { WishListMain } from '../components/src/index';
 import { login } from '../utils/AuthService';
 import { cortexFetch } from '../utils/Cortex';
 import Config from '../ep.config.json';
-
+import WishListMain from '../components/src/WishList/wishlist.main';
```
Comment (dusanradovanovic) [inline src/containers/WishListsPage.tsx:28]: Let's keep the json import after the components.

**Comment by dusanradovanovic** [comment]
I only have a minor comment. I'm so glad this file is gone 😀 Can you please make sure that all tests are passing, just in case 😃

File: src/containers/WishListsPage.tsx (hunk 1 above)
Comment (aChanEP) [reply src/containers/WishListsPage.tsx:28]: Sure thing ill make that change

**Comment by aChanEP** [comment]
> I only have a minor comment. I'm so glad this file is gone 😀 Can you please make sure that all tests are passing, just in case 😃

Sure I can run local tests.  But ive always had the problem where tests fail locally but pass in jenkins

**Comment by shaunmaharaj** [comment]
@aChanEP I can update the Jenkins pipeline to pull from your branch so we can run remotely. Will set that up now before we merge and post back results

**Comment by aChanEP** [comment]
> back

Great thanks!

**Comment by shaunmaharaj** [comment]
Tests passed! All good to merge 🚀

**Pull request merged**